client/main.py

```diff
@@ -13,21 +14,21 @@
 game_win = pg.display.set_mode(res)
 
 with zmq.Context() as zmq_ctx:
-
-    address = loginPage(game_win)
-    should_stop = not SHOULD_STOP_GAME
-
-    while not should_stop:
-
-        player_uuid = uuid.uuid4().hex
-
-        Connection.connect(zmq_ctx, player_uuid, address)
-
-        pg.display.set_caption(f'Playing as: {player_uuid}')
-
-        should_stop = GameSession.run_session(zmq_ctx, game_win, player_uuid, address)
-
-        if should_stop:
-            Connection.disconnect(zmq_ctx, player_uuid, address)
-
-    zmq_ctx.destroy(1000)
+    rejoin = False
```

Comment:
the logic here *works* but it's not well presented. Every line you put down here is one level of tabbing above the original. additionally, you call loginPage twice in the same iteration which shouldn't be the case. Please review what I had originally written for this part